### DISCUSSION TITLE
webERP installation/login Fatal Error

### DISCUSSION THREAD

**Discussion by kaarrtti** [community/community#136208] [Copilot Conversations]
### Select Topic Area

Show & Tell

### Body

Fatal error: Uncaught mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'SET FOREIGN_KEY_CHECKS = 0' at line 1 in C:\xampp\htdocs\weberp\install\index.php:1289 Stack trace: #0 C:\xampp\htdocs\weberp\install\index.php(1289): mysqli_query(Object(mysqli), ' SET names UTF8...') #1 C:\xampp\htdocs\weberp\install\index.php(1165): PopulateSQLDataBySQL('../sql/mysql/co...', Object(mysqli), 'mysqli', 'myweberp', false) #2 C:\xampp\htdocs\weberp\install\index.php(483): PopulateSQLData('../sql/mysql/co...', false, Object(mysqli), 'mysqli', 'myweberp') #3 {main} thrown in C:\xampp\htdocs\weberp\install\index.php on line 1289

**Comment by ghost** [comment]
the same error here 

Fatal error: Uncaught mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'SET FOREIGN_KEY_CHECKS = 0' at line 1 in C:\xampp\htdocs\cms\webERP\install\index.php:1224 Stack trace: #0 C:\xampp\htdocs\cms\webERP\install\index.php(1224): mysqli_query(Object(mysqli), ' SET names UTF8...') #1 C:\xampp\htdocs\cms\webERP\install\index.php(1100): PopulateSQLDataBySQL('../sql/mysql/co...', Object(mysqli), 'mysqli', 'we_weberp', false) #2 C:\xampp\htdocs\cms\webERP\install\index.php(436): PopulateSQLData('../sql/mysql/co...', false, Object(mysqli), 'mysqli', 'we_weberp') #3 {main} thrown in C:\xampp\htdocs\cms\webERP\install\index.php on line 1224

**Comment by walcom** [comment]
up, I am facing the same problem
